README.md

```diff
@@ -67,33 +73,29 @@ If for some reason, you are not able to access your phone, you can use the adb (
 4. Open a terminal and type `adb devices`. This should show your phone. If it does, type `adb install [path-to-apk]`.
 
 ## Manual installation
-This section, is if you don't like appimages, or want to run the game natively. Unless you have a very good reason for it, please use the AppImage instead!
+This section, is if you don't like AppImages, or want to run the game natively. Unless you have a very good reason for it, please use the AppImage instead!
```

Comment:
Should be "Manual installation should not be used without due cause. Unless you have a very good reason for avoiding AppImages, please use the default installation instead!"